command/CommandController.js

```diff
@@ -305,11 +330,26 @@ async function searchAndPlayArticle(res, getBody, searchTerm, summaryOnly) {
     const articleInfo = await searchForPocketArticle(getBody, searchTerm);
     if (articleInfo) {
       console.log(articleInfo);
-      let audioUrl;
-      if (summaryOnly) {
-        audioUrl = await buildSummaryAudioFromUrl(articleInfo.resolved_url);
-      } else {
-        audioUrl = await buildAudioFromUrl(articleInfo.resolved_url);
+
+      // do we already have the audio file?
+      let audioUrl = await helper.getAudioFileLocation(
+        articleInfo.item_id,
+        summaryOnly
+      );
+
+      // if we didn't find it in the DB, create the audio file
+      if (!audioUrl) {
```

Comment:
Does it make sense to make a helper with this and the matching above chunk?

Reply:
These blocks are just slightly different and would require more extensive surgery to make them the same. In the interest of demo prep, I'm going to leave this one alone for now.